### ISSUE TITLE
Branches not showing up

### ISSUE THREAD

**Issue by halleycarleton** [NikhilKalige/git-log#28]
When I run Git Log:Show it only shows one branch, the master. None of my feature or bug branches show up at all. Am I missing something or is this a bug?

**Comment by danielo515** [comment]
I have exacly the same problem. Are you on the master branch?

**Comment by halleycarleton** [comment]
Yeah, I've tried on master as well as a number of other branches.

**Comment by halleycarleton** [comment]
This is an issue on both windows and linux. Currently running Atom 0.174.0.

**Comment by halleycarleton** [comment]
Just reinstalled this package hoping this had been fixed. Unfortunately it is still an issue.
I also noticed that the issue is a little different than I originally thought.

What seems to be happening is that all commits from all branches show up on a single line(visualization of a single branch). So it's not actually showing the master branch, it's just showing all branches as a single unnamed branch.

git-log 0.4.1
Atom 1.12.7
Ubuntu 16.04
